extraction/embedded_tweet_extractor.py

```diff
@@ -0,0 +1,22 @@
+"""This script checks whether DOM has embedded tweet tag or not and
+creates and returns the ETweet object"""
+
+from extraction.data_models.embedded_tweet import ETweet
+from extraction.interface_content_extractor import IContentExtractor
+
+
+class ETweetExtractor(IContentExtractor):
+    """This class inherits IContentExtractor for extracting embedded tweets"""
+
+    def validate_and_extract(self, node):
+        try:
+            if node['class'] == ['twitter-tweet', 'twitter-tweet-rendered'] \
+                    or node['class'] == ['twitter-tweet']:
```

Comment:
Is node['class'] a list? If yes then, can we try this:
`'twitter-tweet' in node['class'] or 'twitter-tweet-rendered' in node['class']`

Because node['class'] can have other elements also.

Reply:
Updated with changes. Thanks for the suggestion!